src/js/Controllers/UIController.js

```diff
@@ -196,7 +196,10 @@ class UIController {
                     rpc.params.buttonName,
                     true
                 ))
-                this.listener.send(RpcFactory.SubscribeButton(rpc.params.appID, rpc.params.buttonName));
+                if(!rpc.params.buttonName || !rpc.params.appID) {
+                    this.listener.send(RpcFactory.ErrorResponse(rpc, 12, "No button provide to subscribe"))
```

Comment:
@LiliiaShlikhtLuxoft `12` is it magic number ? Why 12 ? not 13?

Reply:
@ValeriiMalkov my bad. Fixed